powerauth-java-cmd-lib/src/main/java/io/getlime/security/powerauth/lib/cmd/consts/PowerAuthStep.java

```diff
@@ -93,6 +93,11 @@
      */
     TOKEN_VALIDATE("token-validate", "Token Digest Validation", "validate-token"),
 
+    /**
+     * Send an encrypted request with token-based authentication
+     */
+    TOKEN_ENCRYPT("token-encrypt", "Encrypt Request and Validate Token Digest", "token-encrypt"),
```

Comment:
Please can we have this new value immediately after `TOKEN_CREATE`?

Reply:
I see, the order is alphabetic, moved.